src/config/locales.json

```diff
@@ -67,4 +74,4 @@
 			}
 		]
 	}
-}
+}
```

Comment:
Please fix this. This diff here is unnecessary and the JSON file should include a line break at the end

Reply:
I could make the modifications here on my local, but I don't really understand how git rebase works, so please give me some time (because I'm a Git begginer). I'm so sorry.

Reply:
No worries. If you can't get it to work, then I can fix this myself later, no big deal. I could do this on your branch (which would unnecessarily complicate things for you if you want to or need to fix something again), or I can do it on the master branch after merging.

If you want to give it a try though, do the following:

```sh
# 1. update your local master branch
# assuming "upstream" is the name of the remote repository at "streamlink/streamlink-twitch-gui" (check with git remote -v)
git checkout master
git pull upstream master

# 2. go back to your feature branch and rebase
# do the rebase non-interactively, to not over-complicate things
# assuming your local branch is called "i18n/ja"
git checkout i18n/ja
git rebase master

# 3. fix the issues, stage the changes and amend the previous commit
git add ...
git commit --amend

# 4. force-push the updated local branch to your fork, which will update the pull request
# assuming "github" is the name of your forked remote repository at "akawshi/streamlink-twitch-gui"
git push --force github i18n/ja

```

Reply:
Or just add more commits with the necessary changes and push regularly to your branch. I will then manage this by squash-merging.

Reply:
I am grateful for your many usuful advices. 

I would rebase commits and push -f my remote repository in haste, but I am not confident about these so I will read your advice carefully later and study them. If you wouldn't have time and my change is a mistake, I don't mind you would manage this.